### ISSUE TITLE
Installation help

### ISSUE THREAD

**Issue by DevOpsJon** [mikamai/discourse-doorkeeper-sso#5]
The last part of the installation instructions are not clear to me- this is my first couple days with discourse though. 

You say to clone "the repo" and install it into the plugins directory. But since discourse is installed via Docker now, this won't work since it pulls the latest discourse from their own repo, right? 

Any chance you can update/clarify this part of the instructions for us noobs? 
I saw this post which might be an alternative way to install. Does this work? 
https://meta.discourse.org/t/install-a-plugin/19157

**Comment by chrism2671** [comment]
Yep, I'm having the same issue. I followed the instructions here:
https://meta.discourse.org/t/install-a-plugin/19157/43
in order to install the plugin, however this causes the discourse installation to fail at db:migrate.

**Comment by EMRJ** [comment]
Same issue here - Anyone tell me how to install the plugin?
